README.md

```diff
@@ -15,10 +15,14 @@ pip install rudder-sdk-python
 ## Initializing the RudderStack Client
 
 ```python
-import rudder_analytics
+import rudderstack.analytics as analytics
 
-rudder_analytics.write_key = <SOURCE_WRITE_KEY>
-rudder_analytics.data_plane_url = <DATA_PLANE_URL>
+analytics.write_key = <SOURCE_WRITE_KEY>
```

Comment:
can we provide all the initialization options list and what they do to docs team in order to update the documentation? there is no list anywhere